examples/lessons_api_example.py

```diff
@@ -0,0 +1,55 @@
+# Run with Python 3
+import json
+import requests
+import pylab
+import pandas as pd
+import matplotlib.pyplot as plt
+
+'''This example is made to be simple and useful. 
+It demonstrates how to get lessons data via StepicAPI and why it can be useful.'''
+
+# 1. Get your keys at https://stepic.org/oauth2/applications/ (client type = confidential,
+# authorization grant type = client credentials)
+client_id = "put_yours_here"
+client_secret = "put_yours_here"
+
+# 2. Get a token
+auth = requests.auth.HTTPBasicAuth(client_id, client_secret)
+resp = requests.post('https://stepic.org/oauth2/token/',
+                     data={'grant_type': 'client_credentials'},
+                     auth=auth
+                     )
+token = json.loads(resp.text)['access_token']
```

Comment:
requests has .json() method , can be used here instead of json.loads() , like here - ef7efe2